code/__HELPERS/game.dm

```diff
@@ -53,6 +53,26 @@
 	for(var/I in adjacent_turfs)
 		. |= get_area(I)
 
+/**
+ * Get a bounding box of a list of atoms.
+ *
+ * Arguments:
+ * - atoms - List of atoms. Can accept output of view() and range() procs.
+ *
+ * Returns: list(x1, y1, x2, y2)
+ */
+/proc/get_bbox_of_atoms(list/atoms)
+	var/list/list_x = list()
+	var/list/list_y = list()
+	for(var/atom/A in atoms)
+		list_x += A.x
+		list_y += A.y

```

Comment:
```suggestion
	for(var/a in atoms)
		var/atom/atom_in_list = a
		list_x += atom_in_list.x
		list_y += atom_in_list.y
```
Sorry to keep coming back to this.
If it's guaranteed that the list will contain `atom` elements only then there's no need to check.
To typecast is better performant, as it skips the check  in the loop.